patches/chrome-browser-net-system_network_context_manager.cc.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
+index 2c9b6e9536749bee6c7ca4dbd3c87410e0940f64..4aed95ee28386d5d0ca91a01cd8297d464861227 100644
+--- a/chrome/browser/net/system_network_context_manager.cc
++++ b/chrome/browser/net/system_network_context_manager.cc
+@@ -262,8 +262,10 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
+     DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+     if (!manager_)
+       return;
++    network::ResourceRequest patched = url_request;
```

Comment:
Multiline patches are almost never necessary and should be avoided. These two lines can be replaced by a `define`, but can't we just do `std::move(request), routing_id, request_id, options, brave::OnBeforeSystemRequest(url_request),` where `OnBeforeSystemRequest` returns the patched `network::ResourceRequest`?

Reply:
yeah, good idea - fixed.
Wondering what is the idea behind packing multiline diff into macroses?